x-pack/legacy/plugins/epm/server/lib/kibana/index_pattern/tests/coredns.logs.yml

```diff
@@ -0,0 +1,71 @@
+- name: coredns
```

Comment:
I added this other file, so I could more easily view the snapshot with two different package names.